app/backend/readme.md

```diff
@@ -9,19 +9,19 @@ You can run the whole thing through Docker and docker-compose (see the readme in
 You can run all tests in docker with the following command, executed in the `app` folder:
 
 ```sh
-docker compose -f docker-compose.test.yml up --build
+docker-compose -f docker-compose.test.yml up --build
```

Comment:
`compose` is now a sub-command of `docker`. If you update docker you should be able to do `docker compose`.